platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -422,6 +459,55 @@ public void activateLocationComponent(@NonNull Context context, @NonNull Style s
     applyStyle(options);
   }
 
+  public void activateLocationComponent(@NonNull LocationComponentActivationOptions
+                                          locationComponentActivationOptions) {
+
+    Log.d(TAG, "locationComponentActivationOptions context packageName = " + locationComponentActivationOptions.context().getPackageName());
+    Log.d(TAG, "locationComponentActivationOptions style URL = " + locationComponentActivationOptions.style().getUrl());
+    Log.d(TAG, "locationComponentActivationOptions styleRes = " + locationComponentActivationOptions.styleRes());
```

Comment:
do not log out above and if logging it required use Logger.java instead

Reply:
Was logging as I debugged and played with logic in this method. Removed.